units/crons_event_test.go

```diff
@@ -112,20 +112,17 @@ func (s *cronsEventSuite) SetupTest() {
 }
 
 func (s *cronsEventSuite) TestDegradedMode() {
-	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
-
 	// Reset to original flags after the test finishes.
-	originalFlags, err := evergreen.GetServiceFlags()
+	originalFlags, err := evergreen.GetServiceFlags(s.ctx)
```

Comment:
This switches to using a context that is torn down at the end of the suite rather than the end of the test I think?

I'm guessing that's fine.

Reply:
ah, that's true. I hadn't noticed it.
It seems okay.